src/components/Header.js

```diff
@@ -1,19 +1,39 @@
 import Link from 'next/link';
-import {useRouter} from 'next/router';
+import styled from 'styled-components';
+
+const NaviHeader = styled.header`
+	display: flex;
+	padding: 15px;
+	background-color: #f01878;
+	color: white;
+
+	& a {
+		padding: 0 100px;
+		color: white;
+		text-decoration: none;
+	}
+	& a:hover {
+		color: blanck;
+	}
+	& a:active {
+		color: #4878ff;
+	}
```

Comment:
Unfortunately, this only works with React Router. In Next.js you can change the color via a prop which you can name `active`.

```suggestion
	color: ${({active}) => (active ? '#4878ff' : 'white')};
```

Reply:
- Ok, erledigt :-) Danke.